Apps/W1/HybridGP/app/src/pages/HybridCloudWizardExtension.PageExt.al

```diff
@@ -2,38 +2,30 @@ pageextension 4014 "Hybrid Cloud Wizard Extension" extends "Hybrid Cloud Setup W
 {
     layout
     {
-        addafter(Step5)
+        modify(AllDone)
         {
-            group(GPSpecificSettingsStep)
```

Comment:
Same comments here regarding the deletions, we need to obsolete and make visible false

Reply:
Updated